frontend/coprs_frontend/coprs/templates/groups/user_fas_groups.html

```diff
@@ -37,7 +37,7 @@ <h1>
     <tbody>
     {% for team in teams %}
       <tr>
-        <td><a href="{{ fas_group_href(team) }}">
+        <td><a href="{{ config.GROUP_INFO.link.format(name=team) }}">
```

Comment:
Is this #2563?

Reply:
> Is this #2563?

Don‘t know this ticket before :) but in our case, we should be redirect to openEuler group info page instead of `fas_group` stuff, so i think fas_group_href can be replaced with `GROUP_INFO.link` 😄

Reply:
Yeah, I think that's exactly #2563 (forgotten hard-coded link).  Can you use `Fixes: #2563`, too?